src/AppBundle/Command/AddUserCommand.php

```diff
@@ -193,6 +187,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $encoder = $this->getContainer()->get('security.password_encoder');
         $encodedPassword = $encoder->encodePassword($user, $plainPassword);
         $user->setPassword($encodedPassword);
+        $user->setPlainPassword($plainPassword);
```

Comment:
setting the plain password here for the validator is too late: you already encoded it anyway. The validation should run before